assets/queries/ansible/aws/unprotected_s3_storage/query.rego

```diff
@@ -0,0 +1,43 @@
+package Cx
+
+CxPolicy[result] {
+    document := input.document[i]
+    tasks := getTasks(document)
+    task := tasks[t]
+     
+    object.get(task["amazon.aws.s3_bucket"], "encryption", "undefined") == "undefined"
+
+    
+    result := {
+        "documentId":        document.id,
+        "searchKey":         sprintf("name=%s.{{amazon.aws.s3_bucket}}", [task.name]),
+        "issueType":         "MissingAttribute",
+        "keyExpectedValue":  "amazon.aws.s3_bucket.encryption is set",
+        "keyActualValue": 	 "amazon.aws.s3_bucket.encryption is undefined"
+    }
+}
```

Comment:
According to the documentation:

> Describes the default server-side encryption to apply to new objects in the bucket. In order to remove the server-side encryption, the encryption needs to be set to 'none' explicitly.

So only when we explicitly define as none, is when the encryption is disabled. 
We can cut this part from the query